sdks/python/apache_beam/testing/load_tests/load_test.py

```diff
@@ -18,18 +18,65 @@
 
 import json
 import logging
-import unittest
 
 from apache_beam.metrics import MetricsFilter
 from apache_beam.testing.load_tests.load_test_metrics_utils import MetricsReader
 from apache_beam.testing.test_pipeline import TestPipeline
 
 
-class LoadTest(unittest.TestCase):
-  def parseTestPipelineOptions(self, options=None):
+class LoadTest(object):
+  def __init__(self):
```

Comment:
No particular benefit. Because `LoadTest` is no longer a subclass of `unittest.TestCase`, I had to move this code somewhere. And the reasons why I stopped using `unittest.TestCase` are:
- We had to prevent load tests from being included in other test suites and that's why `LOAD_TEST_ENABLED` guard was used. Now the guard is no longer needed,
- `tearDown()` was actually used to run a pipeline, which was misleading, as tearDown is supposed only to clean up resources after test execution,
- Without nose or pytest, it is difficult to pass extra command-line arguments.

To sum up, I think `unittest` doesn't suit our needs here and can be easily replaced with plain Python classes.